### PR TITLE
BDLaethf27

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/2274

### CODE REVIEW

File: new/LIT6783MarkFive.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6783MarkFive" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Mark 5:1–13</title>
```
Comment (MassimoVilla) [inline new/LIT6783MarkFive.xml:8]: This text is basically a simple biblical quotation to which a supplication has been appended, right? A new work record is to be created for these cases as well? I probably missed a discussion or an issue on this in the past.

File: new/LIT6787PPAyn.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    <relation name="lawd:hasAttestation" active="LIT6787PPAyn" passive="BDLaethf27"/>
+                    <relation name="lawd:hasAttestation" active="LIT6787PPAyn" passive="EMIP00060"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The following is the entire text of the prayer, taken from <ref type="mss" corresp="BDLaethf27"/>.</note>
+                    <ab>
+                        በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩
+                        አምላክ፡ ጸሎት፡ በእንተ፡ ሕማመ፡ ዓይን፡ ወፍልፀተ፡ ርእሰ፡ ክርስቶስ፡ ብርሃን፡ ወምእመን፡ ዝበምራቅከ፡ ከሠትከ፡ አዕይንተ፡ እውራን፡ ወበቃልከ፡ ይትፌውሱ፡ ድውያን፡ ዛቲ፡
```
Comment (MassimoVilla) [inline new/LIT6787PPAyn.xml:65]: ርእስ፡?

File: new/LIT6789HymnRaguel.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                        ሰላም፡ ለከ፡ መልአከ፡ ቅዳሴ፡ ራጉኤል፡ ዘውገ፡ ቅዳስያን፡ እንስሳ፡ ሰዓል፡ ሊተ፡ ሰሬቀ፡ አበሲ፡ ኀበ፡ ወልድ፡ ለምሕረት፡ 
+                            ዘሰብሳ።
+                    </ab>
+                </div>
+                <div type="textpart" xml:lang="gez" subtype="explicit">
+                    <note>The following explicit is taken from <ref type="mss" corresp="BDLaethf27"/>.</note>
+                    <ab>
+                        ሰላም፡ ለከ፡ ምክንያተ፡ ጽሂን፡ ራጉኤል፡ አምጣነ፡ ተሐሥሥ፡ አንተ፡ ከመ፡ አምላክከ፡ በሥጋ፡ ዘሞተ፡ ተወኪሊከ፡ 
+                            ማኅሌ<surplus>ሌ</surplus>ትየ፡ ዘንተ፡ ጸግወኒ፡ ሠናየ፡ ዕሤተ፡ ሰላም፡ ለከ፡
+                        ዘኃይስተ፡ ሰማይ፡ ልዑል፡ ራጉኤል፡ ሊቅ፡  መልአከ፡ ኃይል፡ እስመ፡ አንተ፡ መልአከ፡ ሣህል፡ ሰላም፡ ለከ፡ እግዚኦ፡ አድኅና፡ አምሕማመ፡ ዓይነ፡  ወርቅ፡
```
Comment (MassimoVilla) [inline new/LIT6789HymnRaguel.xml:73]: እምሕማመ፡ (unless it is mistakenly አምሕማመ፡, then I would add `<sic/>`)

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea!

**Comment by thea-m** [comment]
Thank you, Massimo!

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**